feature-libs/user/account/components/login-form/login-form.component.ts

```diff
@@ -17,16 +23,21 @@ import { LoginFormComponentService } from './login-form-component.service';
   standalone: false,
 })
 export class LoginFormComponent {
+  @ViewChild('loginForm') loginForm: ElementRef<HTMLElementTagNameMap['form']>;
+
   constructor(protected service: LoginFormComponentService) {
     useFeatureStyles('a11yPasswordVisibliltyBtnValueOverflow');
   }
 
   form: UntypedFormGroup = this.service.form;
   isUpdating$: Observable<boolean> = this.service.isUpdating$;
+  csrf$ = this.service.csrf$;
+  action = this.service.action;
+  method = this.service.method;
```

Comment:
`medium:` let's consider wrapping this with the feature flag, for better safety.

Reply:
didn't want to introduce feature flag in component. The same logic is covered in the service. If service returns nothing, there will be undefined as expected and not added attribute to html